### ISSUE TITLE
`tidy()` removes NA from model fit, but not from CI.

### ISSUE THREAD

**Issue by sjPlot** [tidyverse/broom#166]
See [this issue](https://github.com/sjPlot/devel/issues/152). If you want to tidy the fitted model attached to this issue, you get an error when `conf.int` is included:

The model summary:

```
> summary(fit)

Call:
lm(formula = Cylinders ~ Origin * Type, data = dat)

Residuals:
    Min      1Q  Median      3Q     Max 
-2.8800 -1.0426  0.0000  0.9574  5.9231 

Coefficients: (3 not defined because of singularities)
                        Estimate Std. Error t value Pr(>|t|)    
(Intercept)               3.6667     0.8187   4.478 9.75e-06 ***
OriginEurope              0.8409     0.5919   1.421  0.15619    
OriginUSA                 0.2338     0.6856   0.341  0.73332    
TypeSedan                 1.3759     0.8317   1.654  0.09882 .  
TypeSports                1.4000     0.8969   1.561  0.11930    
TypeSUV                   2.3333     0.8665   2.693  0.00737 ** 
TypeTruck                 1.8333     0.9600   1.910  0.05688 .  
TypeWagon                 1.2424     0.9236   1.345  0.17933    
OriginEurope:TypeSedan    0.1935     0.6305   0.307  0.75913    
OriginUSA:TypeSedan       0.4348     0.7168   0.607  0.54448    
OriginEurope:TypeSports   0.7011     0.7562   0.927  0.35441    
OriginUSA:TypeSports      2.4773     0.9097   2.723  0.00674 ** 
OriginEurope:TypeSUV      0.3591     0.7949   0.452  0.65171    
OriginUSA:TypeSUV         0.6462     0.7943   0.814  0.41637    
OriginEurope:TypeTruck        NA         NA      NA       NA    
OriginUSA:TypeTruck       0.8912     0.9204   0.968  0.33346    
OriginEurope:TypeWagon        NA         NA      NA       NA    
OriginUSA:TypeWagon           NA         NA      NA       NA    

---
Signif. codes:  0 ‘***’ 0.001 ‘**’ 0.01 ‘*’ 0.05 ‘.’ 0.1 ‘ ’ 1

Residual standard error: 1.418 on 411 degrees of freedom
  (2 observations deleted due to missingness)
Multiple R-squared:  0.1993,    Adjusted R-squared:  0.172 
F-statistic: 7.307 on 14 and 411 DF,  p-value: 1.192e-13
```

This works:

```
> broom::tidy(fit)
                      term  estimate std.error statistic      p.value
1              (Intercept) 3.6666667 0.8187271 4.4784969 9.753051e-06
2             OriginEurope 0.8409091 0.5919387 1.4206017 1.561908e-01
3                OriginUSA 0.2337662 0.6856314 0.3409503 7.333152e-01
4                TypeSedan 1.3758865 0.8316893 1.6543276 9.882441e-02
5               TypeSports 1.4000000 0.8968706 1.5609833 1.192973e-01
6                  TypeSUV 2.3333333 0.8664593 2.6929521 7.371962e-03
7                TypeTruck 1.8333333 0.9600426 1.9096375 5.687581e-02
8                TypeWagon 1.2424242 0.9236487 1.3451264 1.793261e-01
9   OriginEurope:TypeSedan 0.1934608 0.6305281 0.3068234 7.591332e-01
10     OriginUSA:TypeSedan 0.4347917 0.7168174 0.6065585 5.444788e-01
11 OriginEurope:TypeSports 0.7011199 0.7562318 0.9271230 3.544068e-01
12    OriginUSA:TypeSports 2.4773449 0.9097204 2.7231938 6.740858e-03
13    OriginEurope:TypeSUV 0.3590909 0.7949360 0.4517230 6.517068e-01
14       OriginUSA:TypeSUV 0.6462338 0.7943335 0.8135547 4.163711e-01
15     OriginUSA:TypeTruck 0.8912338 0.9204032 0.9683080 3.334600e-01
```

Does not work:

```
> broom::tidy(fit, conf.int = T)
Error in data.frame(..., check.names = FALSE) : 
  arguments imply differing number of rows: 15, 18
```

`confint(fit)` also returns NA values, same as in `summary(fit)` for the coefficients:

```
> confint(fit)
                              2.5 %   97.5 %
(Intercept)              2.05725173 5.276082
OriginEurope            -0.32269599 2.004514
OriginUSA               -1.11401544 1.581548
TypeSedan               -0.25900886 3.010782
TypeSports              -0.36302573 3.163026
TypeSUV                  0.63008866 4.036578
TypeTruck               -0.05387297 3.720540
TypeWagon               -0.57324068 3.058089
OriginEurope:TypeSedan  -1.04600158 1.432923
OriginUSA:TypeSedan     -0.97429400 1.843877
OriginEurope:TypeSports -0.78544473 2.187685
OriginUSA:TypeSports     0.68905961 4.265630
OriginEurope:TypeSUV    -1.20355674 1.921739
OriginUSA:TypeSUV       -0.91522945 2.207697
OriginEurope:TypeTruck           NA       NA
OriginUSA:TypeTruck     -0.91805123 2.700519
OriginEurope:TypeWagon           NA       NA
OriginUSA:TypeWagon              NA       NA
```

[broom-test.zip](https://github.com/dgrtwo/broom/files/529420/broom-test.zip)

**Comment by Zirys1** [comment]
Hey,

is there any solution to this issue yet, or a workaround?
Stata for example "solves" a similar issue in esttab with the "noomitted" option.
Is there anything I can do in order to create a sjt or sjp from a lm-model that has NA's in its coefficient matrix?

**Comment by okwon** [comment]
Hi. I am also suffering from the same issue. Would be super helpful if there is some solution for this.

**Comment by sushmitavgopalan16** [comment]
Hi @alexpghayes ! I would LOVE to work on this next week!

**Comment by alexpghayes** [comment]
Being addressed in #345 at the moment.
